content/en/docs/contribute/participate/roles-and-responsibilties.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Roles and responsibilities
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+Anyone can contribute to Kubernetes. As your contributions to SIG Docs grow, you can apply for different levels of membership in the community. These roles allow you to take on more responsibility within the community. Each role requires more time and commitment. The roles are:
```

Comment:
nit: This text could use some wrapping

Reply:
Purposely left the line wrap out, because it gets in the way of translations. However, I think a nice compromise is adding wrapping at the end of every sentence (so at least the line breaks are logical from a tl perspective)